stage3/class-features.md

```diff
@@ -1,7 +1,6 @@
+# Classes
```

Comment:
Can you separate this one to another PR? So this PR focuses on creating the `stage3` thing and in another PR we seek consensus on the specific class features.

Reply:
I can do, these aren't really material changes to the class features themselves though, just a massaging of the format to more closely resemble other files.

Reply:
I guess you're implicitly advocating for the `stage3/*.md` model as otherwise there is no other change to be made.